lampshade.go

```diff
@@ -330,7 +330,6 @@ type BoundDialer interface {
 // Session is a wrapper around a net.Conn that supports multiplexing.
 type Session interface {
 	net.Conn
-	StatsTracking
```

Comment:
Neither flashlight nor http-proxy-lantern uses `Session` interface directly, so this change won't break API.